packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -157,7 +157,7 @@ export function MediaPlaceholder( {
 			setMedia = ( [ media ] ) => onSelect( media );
 		}
 		mediaUpload( {
-			allowedTypes,
+			allowedTypes: allowedTypes.length > 0 ? allowedTypes : undefined,
```

Comment:
Maybe we can just remove the default value for the prop?

Reply:
That is an alternative. However, it means doing checks at several places that use this variable. I do prefer that method instead of specifying an explicit undefined value. Is this the preference at WordPress?

Reply:
> Maybe we can just remove the default value for the prop?

That is an alternative. However, it means doing checks at several places that use this variable. I do prefer that method instead of specifying an explicit undefined value. Is this the preference at WordPress?

Reply:
I personally prefer removing the default value because what you're doing here is changing the meaning of the prop.

When I pass`allowedTypes = []`, I personally expect that upload is not allowed. Not that all files are allowed.